app/policies/work_version_policy.rb

```diff
@@ -47,17 +47,17 @@ def update?
   #   4. The user is a reviewer of the collection the work is in
 
   def show?
-    depositor? || reviews_collection?
+    depositor? || reviews_collection?(collection)
   end
 
   # The collection reviewers can review a work
 
   def review?
-    record.pending_approval? && reviews_collection?
+    record.pending_approval? && reviews_collection?(collection)
   end
 
   def destroy?
-    (administrator? || depositor?) && record.persisted? && record.version_draft?
+    can_delete_works?(collection) && record.persisted? && record.version_draft?
```

Comment:
Can this first part be `allowed_to?(:destroy?, record.head)` ?

Reply:
I don't think so, because the work policy for deletion is more restrictive than the work_version policy for deletion (since you can delete a work version when its a version draft without also deleting the work itself, but not vice versa).  So there will be some common checks (like who is allowed to do it), but we can't do *all* of the work checks for the work_version.